### ISSUE TITLE
Security radio colour is too dark and too saturated

### ISSUE THREAD

**Issue by sowelipililimute** [DeltaV-Station/Delta-v#5935]
This is a low contrast colour combination with a very saturated colour on a dark background. This makes it hard to read while also being an eyesore. The contrast ratio is 2.54:1, which fails WCAG AA standards.

**Comment by rebe83** [comment]
How's this 

<img width="412" height="230" alt="Image" src="https://github.com/user-attachments/assets/b105ab20-cd2c-4e07-8b30-a7fffb219195" />
